### ISSUE TITLE
Question regarding Pump Flow Rate for Boiler

### ISSUE THREAD

**Issue by joesinger12** [joesinger12/TestTicketTransfer#36]
I’m working on modeling a condenser water system serving water source heat pumps for a high-rise residential project with boilers on the condenser water line providing injection heating – meaning only a fraction of the water is being run through the boilers before being returned and mixed with the rest of the water in the system. The design delta T for my cooling tower is 15 deg which is what I have set for the condenser water fluid system. The delta T through the boiler is 30 deg since it’s only a portion of the water and must mix back in with the system leading to an overall lower delta T. The problem I’m seeing is that the design flow rate for my boiler is being determined by the delta T I set for the condenser water fluid system (15 deg).

This flow rate is twice as much as what has been designed for these pumps due the factor of 2 on the different delta T’s. I’ve had to increase the flow on these pumps to accommodate this so my model will run for now (receiving an error if I don’t). 

In a previous revision of this model I was able to use a lower flow rate on the pumps associated with my boiler without receiving an error (see below).

What would be the reason that this is now causing me an error? Is there a way around having to increase the flow on these boilers and properly model this “injection heating” condition?

Reported by: joesinger12

Original Ticket: [cbecc-com/tickets/2356](https://sourceforge.net/p/cbecc-com/tickets/2356)

**Comment by joesinger12** [comment]
- **summary**: Pump flow rate for boiler --> Question regarding Pump Flow Rate for Boiler
- **assigned_to**: Nikhil Kapur --> Roger Hedrick
- **Type**: Defect --> Other


Original comment by: joesinger12

**Comment by joesinger12** [comment]
RH - CBECC-Com does not directly support the pumping configuration included in your proposed design.  As you have built the model, all the pumps will be operated in parallel.  That said, I am seeing excess operation by the boiler pumps.  These pumps are operating at times when the parent boilers are off.  We will be looking into the cause of this operation.  

The mechanism of specifying the design delta T for the fluid system and using that for sizing child equipment was installed early in the development of CBECC-Com, prior to when use of condenser water loops for WSHPs was included.  It may be appropriate that for such loops, providing separate heating and cooling delta T inputs is appropriate, and we will consider this for possible inclusion in a future update to the software.  

In the meantime, I recommend that you modify the pump head for your boiler pumps such that you get the correct power consumption when the pump is operating, despite the excess flowrate.  You should check the exceptional condition “Are significant modeling approximations used?” and describe your adjustments in the narrative tab, both on the “Project” input screen.  

I did some test runs on another WSHP model.  When I had pumps as children of the tower and boiler, having them all as variable speed seemed to provide the only configuration that had reasonable looking pump operation.  


Original comment by: joesinger12

**Comment by joesinger12** [comment]
- **status**: New --> WontFix


Original comment by: joesinger12

**Comment by joesinger12** [comment]
- **status**: WontFix --> Accepted
- **private**: No --> Yes
- **Milestone**: None --> Com2019.1.0


Original comment by: joesinger12

**Comment by joesinger12** [comment]
- **Milestone**: Com2019.1.0 --> ComDevIssue


Original comment by: joesinger12

**Comment by joesinger12** [comment]
- **Milestone**: ComDevIssue --> CBECC2022DevIssues


Original comment by: joesinger12
